index.html

```diff
@@ -83,240 +91,244 @@ <h4>LET'S CONNECT</h4>
+            </div>
           </div>
         </section>
 
         <section class="canopy-fourth-card-section">
-          <div class="snapshoot-portfolio">
-            <img
-              src="./img/snapshootportfolio(3).svg"
-              alt="snapshoot portfolio"
-            />
-          </div>
-          <div class="project-card">
-            <h2>Multi-Post <br />Stories</h2>
-            <h3>
-              CANOPY &nbsp; <img src="./img/Counter.svg" alt="" /> &nbsp;
-              <span
-                >Back End Dev &nbsp;
-                <img src="./img/Counter.svg" alt="" /> &nbsp;2015</span
-              >
-            </h3>
-            <p>
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
-            <ul>
-              <li>html</li>
-              <li>css</li>
-              <li>Javascript</li>
-            </ul>
-            <button type="button">See Project</button>
+          <div class="mycanopy-section">
+            <div class="project-card">
+              <h2>Uber Navigation</h2>
+              <h3>
+                FACEBOOK &nbsp; <img src="./img/Counter.svg" alt="" /> &nbsp;
+                <span
+                  >Lead Developer &nbsp;
+                  <img src="./img/Counter.svg" alt="" /> &nbsp;2015</span
+                >
+              </h3>
+              <p>
+                A smart assistant to make driving more safe, efficient, and fun
+                by unlocking your most expensive computer: your car.
+              </p>
+              <ul>
+                <li>html</li>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+              </ul>
+              <button type="button">See Project</button>
+            </div>
+            <div class="snapshoot-portfolio">
+              <img
+                src="./img/snapshootportfolio(1).svg"
+                alt="snapshoot portfolio"
+              />
+            </div>
           </div>
         </section>
       </section>
 
-      <section class="about-myself">
-        <section class="skill">
-          <h5>
-            About <br />
-            Myself
-          </h5>
-          <p>
-            Hello I'm a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don't
-            hestiate to contact me.
-          </p>
-
-          <div class="social_media">
-            <h4>LET'S CONNECT</h4>
-            <ul>
+      <section id="about" class="about">
+        <div class="about-container">
+          <div class="about-me">
+            <h2>About <br />Myself</h2>
+            <p>
+              Hello I'm a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If

```

Comment:
- [ ] Kindly stick to the Figma design. 
   - The width of the left section is smaller than the right section in the figma design.
   - The Languages are in one row in the desktop version while they are in one column in the mobile version. You can use either Flexbox or Grid to dynamically change the layout on different screen sizes with the help the media queries that you have already implemented.
   #### Figma
![about-myself-figma](https://user-images.githubusercontent.com/44057863/168441476-09c0e76d-5160-40e2-a87d-0dfe3b449003.png)
   #### Imp
![about-myself-impl](https://user-images.githubusercontent.com/44057863/168441479-bc5efbef-d5a5-458a-889d-dda3578aec86.png)
l

Reply:
Understood